daskdev-sat/app.py

```diff
@@ -30,11 +31,13 @@ def make_cluster():
         scheduler_pod_template = make_pod_from_dict(d)
 
     log.info(f"Starting dask cluster {NAME} in namespace {NAMESPACE}")
+    log.info(f"n_workers: {N_WORKERS}")
     log.info(f"worker config: {WORKER_CONFIG}")
     log.info(f"scheduler config: {SCHEDULER_CONFIG}")
     log.info(f"dashboard address: {DASHBOARD_LINK}")
 
     return SaturnCluster(
```

Comment:
Not related to this PR, but the fact that we have two different but directly related definitions of SaturnCluster (i.e. this one and the one in dask-saturn) was very confusing to me at first glance.

Reply:
yes poor naming on my part. I think this one should have a new name. Maybe KubeCluster, or SpecialKubeCluster?